src/sentry/models/release.py

```diff
@@ -9,26 +9,43 @@
 
 import re
 
-from django.db import models
+from django.db import models, IntegrityError
 from django.utils import timezone
 from jsonfield import JSONField
 
 from sentry.db.models import (
-    BoundedPositiveIntegerField, FlexibleForeignKey, Model, sane_repr
+    BaseModel, BoundedAutoField, BoundedPositiveIntegerField,
+    FlexibleForeignKey, Model, sane_repr
 )
 from sentry.utils.cache import cache
 from sentry.utils.hashlib import md5_text
 
 _sha1_re = re.compile(r'^[a-f0-9]{40}$')
 
 
+class ReleaseProject(BaseModel):
+    __core__ = False
+
+    id = BoundedAutoField(primary_key=True)
+    project = FlexibleForeignKey('sentry.Project', null=True)
+    release = FlexibleForeignKey('sentry.Release')
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_release_project'
+        unique_together = (('project', 'release'),)
+
+
 class Release(Model):
     """
     A release is generally created when a new version is pushed into a
     production state.
     """
     __core__ = False
 
+    organization = FlexibleForeignKey('sentry.Organization', null=True, blank=True)
+    projects = models.ManyToManyField('sentry.Project', related_name='releases',
```

Comment:
yup. this is just step 1. i will follow up with: 
2. data migration to populate organization and the many to many projects field
3. change application code to use `Release.projects` and/or `Release.organization` instead of `Release.project`
4. delete `Release.project` column